### PR TITLE
:children_crossing: Added a scrollbar to the TaskHome view when there are too many tasks

### PR DESCRIPTION
**Pull request by hanskokx**
If you need to scroll to view all the tasks, we now have a scrollbar!

### CODE REVIEW

**Review by Rift3000 — COMMENTED** [review]
I see so the ? was added to ensure null safety was working

**Comment by hanskokx** [comment]
> I see so the ? was added to ensure null safety was working

That means the value _can_ be null. https://dart.dev/null-safety#creating-variables

File: lib/helpers/theme.dart [hunk 1]
```diff
@@ -2,7 +2,7 @@ import 'package:flutter/material.dart';
 
 class CustomTheme {
   static ThemeMode mapThemeMode(String themeMode) {
-    assert(themeMode != null, 'theme mode cannot be null');
+    assert(themeMode != '', 'theme mode cannot be null');
```
Comment (tsonnen) [inline lib/helpers/theme.dart:5]: Not sure this assert is even needed with null safety, The case of it being '' or an unknown value is handled in default case

File: pubspec.yaml [hunk 2]
```diff
@@ -1,20 +1,10 @@
...
-# In Android, build-name is used as versionName while build-number used as versionCode.
-# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
-# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
-# Read more about iOS versioning at
-# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
```
Comment (tsonnen) [inline pubspec.yaml:7]: WOOO Null Safety 💪

**Review by tsonnen — COMMENTED** [review]
Looks good

File: lib/helpers/theme.dart (hunk 1 above)
Comment (hanskokx) [reply lib/helpers/theme.dart:5]: Because that's what the auto-migration did 🤷🏻‍♂️

**Pull request merged**